packages/client3/components/design-system/avatar/avatar.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { Avatar } from ".";
+
+export default {
+  title: "FAD/Components/Avatar",
+  component: Avatar,
+} as ComponentMeta<typeof Avatar>;
+
+export const AvatarStory: ComponentStory<typeof Avatar> = (args) => {
+  return <Avatar {...args} />;
+};
```

Comment:
Doesn't look like this works with Storybook. I would suggest having a Storybook avatar (similar to the logo story) where it imports an svg. Something like: 
```

import AvatarSvg from "./Avatar.svg";

export function AvatarStoryBook() {
  return <AvatarSvg />;
}
```

Then having that displayed with `classNames` passed to it to adjust height/width.